simpleclient/src/main/java/io/prometheus/client/SimpleCollector.java

```diff
@@ -108,6 +108,17 @@ protected void initializeNoLabelsChild() {
     }
   }
 
+  /**
+   * Return the child with no labels, which should have been already initialised. It is an error to call this method
+   * if the collector is labelled.
+   */
+  protected Child getNoLabelsChild() throws IllegalArgumentException {
```

Comment:
IllegalArgumentException is a runtime exception, no need to declare it.

Reply:
> IllegalArgumentException is a runtime exception, no need to declare it.

That's debatable, but since this is your library I'll go along with your preference :-)